angryFace.txt

```diff
@@ -0,0 +1 @@
+Angry Face
```

Comment:
Hey, consider using the actual angry face emoji.  if you are on a windows machine

simple use "windows key + '.'" and this will bring up the emoji window, from there, select the angry emoji